cache/in-memory/src/model/member.rs

```diff
@@ -102,8 +102,8 @@ impl PartialEq<Member> for CachedMember {
     }
 }
 
-impl PartialEq<&PartialMember> for CachedMember {
-    fn eq(&self, other: &&PartialMember) -> bool {
+impl PartialEq<PartialMember> for CachedMember {
```

Comment:
These changes are unrelated to the name of the PR. If you wish to make them, please make them in a separate PR. We try to separate concerns of each PR so that if the change causes a problem we can easily roll it back.